csv/ParameterDefs.csv

```diff
@@ -20,9 +20,9 @@ DOC:CI_Attribute_Name,,Parameter.name,[X],Parameter.parameter_type,,Parameter.va
 BASE,,conductivity,PD1,quantity,,float32,,S m-1,-9999999,Seawater Conductivity,6,,,,CONDWAT,sea_water_electrical_conductivity,CONDWAT_L1,"Seawater conductivity refers to the ability of seawater to conduct electricity. The presence of ions in the seawater, such as salt, increases the electrical conducting ability of seawater. As such, conductivity can be used as a proxy for determining the quantity of salt in a sample of seawater. Units: S m-1",Seawater Electrical Conductivity,Science Data,L1
 BASE,,pressure,PD2,quantity,,float32,,dbar,-9999999,Seawater Pressure,3,,,,PRESWAT,sea_water_pressure,PRESWAT_L1,Seawater Pressure refers to the pressure exerted on a sensor in situ by the weight of the column of seawater above it. It is calculated by subtracting one standard atmosphere from the absolute pressure at the sensor to remove the weight of the atmosphere on top of the water column. The pressure at a sensor in situ provides a metric of the depth of that sensor. Units: dbar,Seawater Pressure,Science Data,L1
 BASE,,salinity,PD3,quantity,,float32,,1,-9999999,Practical Salinity,3,,,,PRACSAL,sea_water_practical_salinity,PRACSAL_L2,Salinity is generally defined as the concentration of dissolved salt in a parcel of seawater. Practical Salinity is a more specific unitless quantity calculated from the conductivity of seawater and adjusted for temperature and pressure. It is approximately equivalent to Absolute Salinity (the mass fraction of dissolved salt in seawater) but they are not interchangeable. Units: unitless,Seawater Practical Salinity,Science Data,L2
-BASE,,temperature,PD4,quantity,,float32,,deg_C,-9999999,Seawater Temperature,4,,,,TEMPWAT,sea_water_temperature,TEMPWAT_L1,Seawater temperature near the sensor. Units: degrees Celsius,Seawater Temperature,Science Data,L1
+BASE,,temperature,PD4,quantity,,float32,,ºC,-9999999,Seawater Temperature,4,,,,TEMPWAT,sea_water_temperature,TEMPWAT_L1,Seawater temperature near the sensor. Units: degrees Celsius,Seawater Temperature,Science Data,L1
```

Comment:
When updating the units field, please also remove the `Units ...` from the description field.